docs/reference/release-notes/highlights.asciidoc

```diff
@@ -43,19 +43,154 @@ Newly created indices abstracted by a <<data-streams,data stream>> will be alloc
+CircuitBreakingException is thrown if exceeds the maximum memory allowed in this
+breaker.
+
+This size is estimated as roughly 1.5 times the size of the serialized
+aggregations that need to be reduced. This estimation can be completely off for
+some aggregations but it is corrected with the real size after the reduce
+completes. If the reduce is successful, we update the circuit breaker to remove
+the size of the source aggregations and replace the estimation with the
+serialized size of the newly reduced result.
+
+[discrete]
+[[eql-case-sensitivity-operator]]
+=== EQL: Case-sensitivity and the `:` operator
+
+In 7.10, we made most EQL operators and functions case-sensitive by default.
+We've also added `:`, a new case-insensitive equal operator. Designed for
+security use cases, you can use the `:` operator to search for strings in
+Windows event logs and other event data containing a mix of letter cases.
+
+[source,console]
+----
+GET /my-index-000001/_eql/search
+{
+  "query": """
+    process where process.executable : "c:\\\\windows\\\\system32\\\\cmd.exe"
+  """
+}
+----
+// TEST[setup:sec_logs]
+
+For more information, see the {ref}/eql-syntax.html[EQL
+syntax documentation].
+
+[discrete]
+[[deprecate-rest-api-access-to-system-indices]]
+=== REST API access to system indices is deprecated
+
+We are deprecating REST API access to system indices. Most REST API requests
+that attempt to access system indices will return the following deprecation
+warning:
+
+[source,text]
+----
+this request accesses system indices: [.system_index_name], but in a future
+major version, direct access to system indices will be prevented by default
+----
+
+The following REST API endpoints access system indices as part of their
+implementation and will not return the deprecation warning:
 
-// Omit the notable highlights tag for entries that only need to appear in the ES ref:
-// [discrete]
-// === Heading
-//
-// Description.
+* `GET _cluster/health`
+* `GET {index}/_recovery`
+* `GET _cluster/allocation/explain`
+* `GET _cluster/state`
+* `POST _cluster/reroute`
+* `GET {index}/_stats`
+* `GET {index}/_segments`
+* `GET {index}/_shard_stores`
+* `GET _cat/[indices,aliases,health,recovery,shards,segments]`
+
+[discrete]
+[[add-system-read-thread-pool]]
+=== New thread pools for system indices
+
+We've added two new thread pools for system indices: `system_read` and
+`system_write`. `system_read` is a `fixed` thread pool used to manage resources for
+read operations targeting system indices. Similarly, `system_write` is a
+`fixed` thread pool used to manage resources for write operations targeting
+system indices. Both have a maximum number of threads equal to `5`
+or half of the available processors, whichever is smaller.
+
+[discrete]
+[[metadata-flag-system-indices]]
+=== New metadata flag for system indices
+
+To prevent performance issues related to searches that target
+system indices, we've added a new metadata flag to mark system and non-system
```

Comment:
How does adding a flag help prevent performance issues?

Reply:
This is a good question. I think the answer is that we prevent requests that target system indices, which could inadvertently slow searches.

That said, I think this may be better combined with the `REST API access to system indices is deprecated` section.